apps/paymaster-proxy/package.json

```diff
@@ -21,5 +21,8 @@
   },
   "dependencies": {
     "express": "^4.18.2"
-  }
+  },
+  "files": [
+    "dist"
+  ]
```

Comment:
This is required for `pnpm deploy` to include the built `dist` folder, as it currently doesn't include files included in a packages's local `.gitignore`

context: https://github.com/pnpm/pnpm/issues/7286